docs/getting-started.md

```diff
@@ -419,3 +428,75 @@ if (messageBack.body.type === MessageBodyType.SUBMIT_ATTESTATION_FOR_CLAIM) {
   const myAttestedClaim = messageBack.body.content
 }
 ```
+
+## 6. Verify a claim
+
+As in the attestation, you need a second identity to act as the verifier:
+
+```typescript
+const verifier = await Kilt.Identity.buildFromMnemonic()
+```
+
+Before a claimer sends any data to a verifier, the verifier needs to initiate the verification process by requesting a presentation for a specific CTYPE.
+Therefore, the verifier knows which properties are included in the attested claim and can request to see any combination of these publicly (including all of them or none).
+This is an **important feature for the privacy of a claimer** as this enables them to only show necessary properties for a specific verification.
+However, a verifier has to **allow privacy enhancement** for this to work.
+Otherwise, the verifier will always see all properties of the claim independent of the requested ones.
```

Comment:
This confused me a bit. We can choose to exclude specific properties even if we use the public credentials. (although this provides very little privacy)

Reply:
Are you sure about that? When I run the `getting-started.ts` with no requested properties, the verifier still receives the claim object with all properties.

Reply:
As discussed, I will remove these lines as this is unclear/TBD with the optional portablegabi PR.